requirements-dev.txt

```diff
@@ -1,17 +1,10 @@
 build==0.7.0
 coverage
 mock
-nose
-nose-no-network
-nose-timer
 pycodestyle==2.6.0
 pyflakes==2.2.0
+pytest
+pytest-network
```

Comment:
no version pinning neede?

Reply:
I left it unpinned since nose wasn't before. I do plan to update all the requirements (they're pretty stale at this point) in the near future and I'll probably go with pinning everything then like I have in the provider specific repos.